infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/test/dialect/impl/counter/LocalCacheNextValueGenerationTest.java

```diff
@@ -52,41 +41,15 @@ public void testExceptionWithSequencesGenerator() throws Exception {
 		thrown.expect( HibernateException.class );
 		thrown.expectMessage( "OGM001108: Sequences or di generation is not supported for local caches" );
 
-		startAndCloseFactory( EntityWithSequence.class );
+		startAndCloseFactory( EntityWithSequence.class, "infinispan-local.xml"   );
```

Comment:
double spaces.

I don't think I told you before but we have a project with template and formatters for the Hibernate code style: https://github.com/hibernate/hibernate-ide-codestyles